brainglobe_utils/qtpy/logo.py

```diff
@@ -6,18 +6,51 @@
 def _docs_links_widget(
     package_name: str,
     package_tagline: str,
-    tutorial_file_name: str,
+    tutorial_file_name: str = None,
+    documentation_path: str = None,
+    citation_doi: str = None,
+    github_repo_name: str = None,
+    help_text: str = None,
```

Comment:
```suggestion
    tutorial_file_name: Optional[str] = None,
    documentation_path: Optional[str] = None,
    citation_doi: Optional[str] = None,
    github_repo_name: Optional[str] = None,
    help_text: Optional[str] = None,
```

Will also need to `from typing import Optional` at the top of the file.